domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -76,6 +78,20 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  fun createNumberWithUnits(number: Fraction, units: List<NumberUnit>):
```

Comment:
Rename this to : `createNumberWithUnitsForFraction` considering that this function only sets a fraction.

Also, it might be a nice idea to introduce other functions and write tests for them too in the corresponding test file.

Reply:
Number with units can be of two forms:

1. `5 km/hr`
2. `8 5/3 hours`

In (1.) the value is a real number only and in (2.) the value is real + fraction. 

This particular function only tests the fraction part and not the real part. 

I suggest avoiding the previous comment and rewriting this function something like this:

`fun createNumberWithUnits(real: Real, fraction: Fraction, units: List<NumberUnity>): `

In this function you can pass any types of details.

Reply:
The proto had oneof so I though either a fraction value or a real, and If the third condition is also true then three functions would be needed, Because I don't think we can take optional parameters, I mean that if we are creating a function with three arguments we have to pass three values to it.

Reply:
Okay, then in that case we need to functions:

`createNumberWithUnitsForFraction` and `createNumberWithUnitsForReal`

Reply:
Ok. I will update the functions and more tests for each. Thanku

Reply:
Done.

Reply:
Thanks for updating this.